src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskListTabPanel.tsx

```diff
@@ -144,7 +144,11 @@ const TaskListTabPanel = (props: TaskListTabPanelProps) => {
     [assignees]
   );
 
-  if (isLoading || assigneeIsLoading || !auth.user || !team) return <LoadingIndicator />;
+  const failMessageStyle: SxProps = { py: '25px', textAlign: 'center' };
+
+  if (isLoading || assigneeIsLoading || !auth.user) return <LoadingIndicator />;
+  if (!team)
+    return <>{value === index && <Typography sx={failMessageStyle}>No team assigned to this project!</Typography>}</>;
```

Comment:
id say just put the sx inline since it's only getting used once

also I think a more informational message like "A project can only have tasks if it is assigned to a team"